src/resolvers/helpers/__tests__/filter-test.js

```diff
@@ -50,6 +50,7 @@ describe('Resolver helper `filter` ->', () => {
       expect(args.filter.type).toBeInstanceOf(InputTypeComposer);
     });
 
+    // TODO: use _operators[_id][in] ? why have _ids in root?
     it('should return filter with field _ids', () => {
```

Comment:
Why add this virtual field when we have operators[_id][in] thats useable?

Reply:
Really I don't remember.
Probably it was created for `findByIds`, `removeByIds` resolvers.